contributing/code/conventions.rst

```diff
@@ -77,6 +77,29 @@ must be used instead (where ``XXX`` is the name of the related thing):
     "replaceXXX", on the other hand, cannot add new elements. If an unrecognized
     key is passed to "replaceXXX" it must throw an exception.
 
+.. _contributing-code-conventions-comparisons:
+
+Comparisons
+-----------
+
+Use `identical comparison`_ when the expected value must match a specific type:
```

Comment:
Actually, strict comparison should be used everywhere *except* when you really need type juggling (in most cases, you don't want it)